interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -112,14 +120,30 @@ private void parseArgs(String[] args) {
       }
     }
 
+    if (secureMode == (port == maintenancePort)) {
+      System.err.println(
+          "port and maintenancePort should be same for insecure and different for secure mode: port="
+              + port
+              + ", maintenancePort="
+              + maintenancePort);
+      usage = true;
+    }
+
     if (usage) {
       XdsTestServer s = new XdsTestServer();
       System.err.println(
           "Usage: [ARGS...]"
               + "\n"
-              + "\n  --port=INT          listening port for server."
+              + "\n  --port=INT          listening port for test server."
               + "\n                      Default: "
               + s.port
+              + "\n  --maintenancePort=INT      listening port for other servers."
```

Comment:
I think these scripts already do `snake_case` for args: https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md#client
```suggestion
              + "\n  --maintenance_port=INT      listening port for other servers."
```

Same for `secureMode` -> `secure_mode`

Reply:
done